pkg/metronome/client.go

```diff
@@ -0,0 +1,54 @@
+package metronome
+
+import (
+	"encoding/json"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+)
+
+
+// Constants for embed query parameters
+const embedActiveRuns = "activeRuns"
+const embedSchedules = "schedules"
+const embedHistory = "history"
```

Comment:
This is unexported and unused.

Reply:
True, but this will be used in one of the other methods in this file.